### PR TITLE
[DOCS] Improving code snippets for quantization

### PR DESCRIPTION
**Pull request by sgolebiewski-intel**
Improving code snippets for `Quantize a Model` section in the [article](https://docs.openvino.ai/nightly/basic_quantization_flow.html#quantize-a-model)
This PR addresses the following JIRA ticket: 111579

### CODE REVIEW

File: docs/optimization_guide/nncf/ptq/code/ptq_onnx.py [hunk 1]
```diff
@@ -16,7 +16,8 @@ def transform_fn(data_item):
 #! [dataset]
 
 #! [quantization]
-model = ... # onnx.ModelProto object
+import onnx
+model = onnx.ModelProto() # onnx.ModelProto object
```
Comment (alexsu52) [inline docs/optimization_guide/nncf/ptq/code/ptq_onnx.py:20]:
```suggestion
import onnx
model = onnx.load("model path")
```

I hope it is more readable.

File: docs/optimization_guide/nncf/ptq/code/ptq_openvino.py [hunk 2]
```diff
@@ -15,7 +15,8 @@ def transform_fn(data_item):
 #! [dataset]
 
 #! [quantization]
-model = ... # openvino.runtime.Model object
+import openvino as ov
+model = ov.Model() # openvino.runtime.Model object
```
Comment (alexsu52) [inline docs/optimization_guide/nncf/ptq/code/ptq_openvino.py:19]:
```suggestion
import openvino.runtime as ov
model = ov.Core().read_model("model path")
```

remove `import openvino.runtime as ov` from the 25 line

File: docs/optimization_guide/nncf/ptq/code/ptq_tensorflow.py [hunk 3]
```diff
@@ -15,7 +15,8 @@ def transform_fn(data_item):
 #! [dataset]
 
 #! [quantization]
-model = ... # tensorflow.Module object
+import tensorflow as tf
+model = tf.Module() # tensorflow.Module object
```
Comment (alexsu52) [inline docs/optimization_guide/nncf/ptq/code/ptq_tensorflow.py:19]:
```suggestion
model = tf.saved_model.load("model_path")
```

File: docs/optimization_guide/nncf/ptq/code/ptq_torch.py [hunk 4]
```diff
@@ -15,7 +15,8 @@ def transform_fn(data_item):
 #! [dataset]
 
 #! [quantization]
-model = ... # torch.nn.Module object
+import torch.nn as nn
+model = nn.Module() # torch.nn.Module object
```
Comment (alexsu52) [inline docs/optimization_guide/nncf/ptq/code/ptq_torch.py:19]: PyTorch model can not be loaded from the file, thus I would suggest to stay the previous version.

**Review by alexsu52 — CHANGES_REQUESTED** [review]
Users, usually, copy code from examples and try to run it. Did you try to run code of your changes?

File: docs/optimization_guide/nncf/ptq/code/ptq_torch.py (hunk 4 above)
Comment (sgolebiewski-intel) [inline docs/optimization_guide/nncf/ptq/code/ptq_torch.py:19]: How about such an example: 

```suggestion
import torchvision
model = torchvision.models.resnet50(pretrained=True)
```
Would it be viable in this context then?

Comment (alexsu52) [reply docs/optimization_guide/nncf/ptq/code/ptq_torch.py:19]: I don't mind

**Review by alexsu52 — APPROVED** [review]
LGTM

**Review by zKulesza — APPROVED** [review]

**Pull request merged**